libbpf-tools/tcpretrans.h

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Anton Protopopov
+// Copyright (c) 2021 Red Hat, Inc.
+#ifndef __TCPRETRANS_H
+#define __TCPRETRANS_H
+
+#define RETRANSMIT  1
+#define TLP         2
+
+struct event {
+	union {
+		__u32 saddr_v4;
+		__u8 saddr_v6[16];
+	};
+	union {
+		__u32 daddr_v4;
+		__u8 daddr_v6[16];
+	};
```

Comment:
Since we use same struct for v4/v6 event, this can be a single field like `__u8 saddr[16]`.
In user space, `inet_ntop` can handle this correctly.

Reply:
I went ahead and attempted this, seems to work locally, let me know what you think.